tests/run-tests

```diff
@@ -5,21 +5,29 @@ import subprocess
 import sys
 import platform
 import argparse
+import inspect
 import re
 from glob import glob
 
+# See stackoverflow.com/questions/2632199: __file__ nor sys.argv[0]
+# are guaranteed to always work, this one should though.
+BASEPATH = os.path.dirname(os.path.abspath(inspect.getsourcefile(lambda: None)))
+
+def base_path(*p):
+    return os.path.join(BASEPATH, *p).replace('\\', '/')
```

Comment:
Would it be safer/better to return and absolute/canonical path here?

Reply:
You mean to strip out `..` etc? Yes that's probably better

Reply:
yes, that is what I mean

Reply:
Ok changed that.